allensdk/test_utilities/custom_comparators.py

```diff
@@ -8,7 +8,7 @@ class WhitespaceStrippedString(object):
     whitespace. By default removes any unicode whitespace character that
     matches the regex \s, (which includes [ \t\n\r\f\v], and other unicode
     whitespace characters).
-    """
+    """ # noqa W605
```

Comment:
Why do we need the `W605`? When I looked it up it seems related to an invalid escape sequence. But I don't see any escape sequences here

Reply:
Due to the `/s`

Reply:
Heads up: I ran into the same thing on `master` and took a different tack

https://github.com/AllenInstitute/AllenSDK/commit/ba86d3a43fc559df223cb87cc7c68be51dc0825e

Probably won't result in a merge conflict when the time comes.